### ISSUE TITLE
Tabs widget visual states are indistinct

### ISSUE THREAD

**Issue by dengeist** [w3c/aria-practices#1339]
## Problem
The tabs widget examples both add a band of color to the top of the active tab. 
<img width="330" alt="" src="https://user-images.githubusercontent.com/13525251/74792238-b504e380-5271-11ea-8972-cc0f897c6050.png">

That same band is recolored and _slightly_ shortened when the tab receives focus.
<img width="330" alt="Screen Shot 2020-02-18 at 5 11 49 PM" src="https://user-images.githubusercontent.com/13525251/74792262-c5b55980-5271-11ea-9dea-0668080353e1.png">

This change is almost color-only in nature, and doesn't make the state of the tab glanceable. It also means that a user could forget whether a color means "active" or "in focus". Colorblind users may not be able to perceive the distinction.

## Proposal
Use distinct visual affordances for the active and focused states of a tab button. Some examples:
- [Tabs widget used by Pearson](https://pearson-ux.github.io/web-components/tab-navigation/)
- [Tabs widget proposed by Inclusive Components](https://codepen.io/heydon/pen/veeaEa)

**Comment by JAWS-test** [comment]
The linked examples both violate WCAG 2.1 SC 1.4.11 because the contrasts of focus or marking of selected are insufficient (1.5:1 for focus by Heydon, 2.6:1 for selected by Pearson). The APG example, however, is sufficient according to your illustrations, because the focus is visible through a focus frame. The colored bar is additional information. This is only valid for Firefox, the focus frame is not shown in Chrome. The WCAG 2.1 SC 2.4.7 is fulfilled by the resizing the colored bar, but the example will probably violate the increased contrast requirements of WCAG 2.2. I therefore support the proposal to improve the focus visibility

**Comment by carmacleod** [comment]
Agree. The mouse hover uses the same styling as focus (slightly thinner red band, instead of thin blue band), and I have had the disconcerting feeling of "what just happened??" when moving the mouse around in this example. These Tabs examples need a good visual design makeover. @ZoeBijl, are you up for this? :)

**Comment by carmacleod** [comment]
That would be awesome, @dengeist! Pretty sure @ZoeBijl wouldn't mind at all! ;)
Thank-you!
